atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1299,6 +1299,8 @@ void WebContents::OnCursorChange(const content::WebCursor& cursor) {
   if (cursor.IsCustom()) {
     Emit("cursor-changed", CursorTypeToString(info),
       gfx::Image::CreateFrom1xBitmap(info.custom_image),
+      gfx::Rect(info.custom_image.width(), info.custom_image.height()),
+      info.hotspot,
```

Comment:
The new arguments should be placed after `info.image_scale_factor`, otherwise it would be a breaking change. And can you mention the new arguments in `docs/web-contents.md`?